SEBrowser/Scripts/TSX/global.d.ts

```diff
@@ -49,9 +49,21 @@ export namespace Redux {
         Asset: IGenericSliceState<SystemCenter.Types.DetailedAsset>,
         AssetGroup: IGenericSliceState<XDA.Types.AssetGroup>,
         Location: IGenericSliceState<SystemCenter.Types.DetailedLocation>,
-        EventNote: NoteState,
         Settings: SettingsState,
-        EventType: IGenericSliceState<SEBrowser.EventType>
+        EventType: IGenericSliceState<SEBrowser.EventType>,
+        EventNote: IGenericSliceState<XDA.Types.Note>,
+        MeterNote: IGenericSliceState<XDA.Types.Note>,
+        AssetNote: IGenericSliceState<XDA.Types.Note>,
+        LocationNote: IGenericSliceState<XDA.Types.Note>,
+    }
+
+
+    interface NoteState {
+        Status: Application.Types.Status,
+        Data: OpenXDA.Types.Note[],
+        SortField: keyof OpenXDA.Types.Note,
+        Ascending: boolean,
+        ParentID: (number | null),
     }

```

Comment:
```suggestion
```

This appears to only be used in EventNoteSlice.ts which this PR seems to make unused now. I suggest to remove both this interface and that file and simply restore from git if it is needed again in the future.